README.md

```diff
@@ -39,8 +39,6 @@ Leave it empty or remove it to not get notified via Slack.
 variable empty or remove it to use the public `lcd.terra.dev` lcd server.
 
 Optionally set
-- `ALLOWED_USER_IDS` comma separated list of Telegram ids of users allowed to vote.
-Example of usage: `ALLOWED_USER_IDS=123,456` - users with ids `123` and `456` are allowed to perform transactions.
```

Comment:
We don't need `allowed_users` anymore because now it is supposed that people use our public bot to delegate voting, we check in code if user clicking button has its voting delegated